### ISSUE TITLE
Markdown-it rules in _config.ts cause TypeError

### ISSUE THREAD

**Issue by kari** [lumeland/lume#259]
I was trying to overwrite [`markdown-it-footnote` template rules](https://github.com/markdown-it/markdown-it-footnote#customize) in `_config.ts`:

```
const markdown = {
  plugins: [markdown_footnote],
  rules: {
    "footnote_block_open": () => (
    '<footer class="footnotes">\n' +
    '<ol class="footnotes-list">\n'
    )
  },
};
```

However, this caused a `TypeError: Cannot set properties of undefined (setting 'footnote_block_open') at (https://deno.land/x/lume@v1.11.1/plugins/markdown.ts:91:25)` when building the site.

Looking at [markdown-it documentation](https://github.com/markdown-it/markdown-it/blob/master/docs/architecture.md#renderer), it seems that in markdown.ts:91 instead of `engine.rules[name] = rule;`, should it be `engine.renderer.rules[name] = rule;`? After doing this change, my site built successfully and used the customized `footnote_block_open`. However, I'm not too familiar with lume or markdown-it to say if this is a correct fix?

**Comment by oscarotero** [comment]
Hi!
Yes, definitely this is a Lume bug, thanks for reporting.
I'm going to fix it now.

**Comment by oscarotero** [comment]
Hi again @kari 
It's fixed in Lume v1.11.4.

**Comment by kari** [comment]
Thanks for the fix!
